python/inc/bond/python/struct.h

```diff
@@ -26,14 +26,14 @@ class def_readwrite_property
         : c(c)
     {}
 
-    template <typename Field>
-    void operator()(const Field&) const
+    template <uint16_t field_id, typename ModifierTag, typename Struct, typename FieldType, FieldType Struct::*field_ptr, const bond::Metadata* metadata_ptr>
```

Comment:
Was this change required to get this to build?

* If so, then the introduction of the anonymous struct is not working as I'd expect it to (the anonymous struct should have an _is a_ relationship with `FieldTemplate`), and is a large breaking change.
* If this was just done to clean up the code, that's fine, but I need to get confirmation from you.